From 0c196a184f5fc688f54c766cdfec24715927f992 Mon Sep 17 00:00:00 2001
From: Markus Niebel <Markus.Niebel@tq-group.com>
Date: Fri, 9 Aug 2019 12:17:09 +0200
Subject: [PATCH] tqma8mx: consolidate lpddr configs

- remove unused file
- rename remainig to reflect their purpose for HW rev.010x

Signed-off-by: Markus Niebel <Markus.Niebel@tq-group.com>
---
 board/tqc/tqma8mx/Makefile                                   | 5 ++---
 ...{lpddr4_timing_1g.c => tqma8mx-1gb-lpddr4_timing_r010x.c} | 0
 ...lpddr4_timing_2gs.c => tqma8mx-2gb-lpddr4_timing_r010x.c} | 0
 3 files changed, 2 insertions(+), 3 deletions(-)
 rename board/tqc/tqma8mx/{lpddr4_timing_1g.c => tqma8mx-1gb-lpddr4_timing_r010x.c} (100%)
 rename board/tqc/tqma8mx/{lpddr4_timing_2gs.c => tqma8mx-2gb-lpddr4_timing_r010x.c} (100%)

diff --git a/board/tqc/tqma8mx/Makefile b/board/tqc/tqma8mx/Makefile
index 26d659474a..4ec738a9de 100644
--- a/board/tqc/tqma8mx/Makefile
+++ b/board/tqc/tqma8mx/Makefile
@@ -10,8 +10,7 @@ obj-$(CONFIG_MBA8MX) += tqma8mx-mba8mx.o
 
 ifdef CONFIG_SPL_BUILD
 obj-y += spl.o
-obj-y += lpddr4_timing_2gs.o
-obj-y += lpddr4_timing_2gm.o
-obj-y += lpddr4_timing_1g.o
+obj-y += tqma8mx-2gb-lpddr4_timing_r010x.o
+obj-y += tqma8mx-1gb-lpddr4_timing_r010x.o
 obj-$(CONFIG_MBA8MX) += spl-mba8mx.o
 endif
diff --git a/board/tqc/tqma8mx/lpddr4_timing_1g.c b/board/tqc/tqma8mx/tqma8mx-1gb-lpddr4_timing_r010x.c
similarity index 100%
rename from board/tqc/tqma8mx/lpddr4_timing_1g.c
rename to board/tqc/tqma8mx/tqma8mx-1gb-lpddr4_timing_r010x.c
diff --git a/board/tqc/tqma8mx/lpddr4_timing_2gs.c b/board/tqc/tqma8mx/tqma8mx-2gb-lpddr4_timing_r010x.c
similarity index 100%
rename from board/tqc/tqma8mx/lpddr4_timing_2gs.c
rename to board/tqc/tqma8mx/tqma8mx-2gb-lpddr4_timing_r010x.c
